spec/views/layouts/application.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+# Capybaraを使うためにsystem specを指定する
+RSpec.describe "layouts/application", type: :system do
+  before do
+    visit root_path
+  end
+
+  describe "letter_opener link" do
+    context "when production environment" do
+      before do
+        allow(Rails).to receive(:env).and_return(ActiveSupport::StringInquirer.new("production"))
+      end
+
+      it "is not in production and test environment" do
```

Comment:
```suggestion
      it "doesn't have the link to letter_opener" do
```

context:  本番環境の場合
it:  レターオープナーへのリンクを持ってない
みたいな英文が自然だと思う。

Reply:
これは主語はdescribeで述べられている「letter_opener link」。
なので、提案された文章では「リンクがリンクを持たない」という意味になってしまう。
なので「リンクが存在しない」という意味の元の文がいい。

Reply:
では、
it doesn't exist in production environment
がいいと思います。

Reply:
理由
- 「本番環境では〜」はすでにcontextで書いているのでitに要らない
- letter_opener link is not だと「レターオープナーのリンクではない」という意味になると思う

Reply:
OK
じゃあ it does not exist でいいね

Reply:
> letter_opener link is not だと「レターオープナーのリンクではない」という意味になると思う

これはbe notだから存在しないという意味が近くて、リンクではない、ではないと思う。
強いて言えば、「レターオープナーのリンクは違う」のほうが正しいんじゃないかな

Reply:
commit 90a8561e186006ad308ba58d0b3b13a139bc4896  で対応